examples/albert/run_trainer.py

```diff
@@ -246,19 +247,18 @@ def main():
 
     adjusted_target_batch_size = collaboration_args.target_batch_size - collaboration_args.batch_size_lead
 
-    collaborative_optimizer = hivemind.CollaborativeOptimizer(
-        opt=opt,
+    collaborative_optimizer = Optimizer(
         dht=dht,
-        scheduler=scheduler,
-        prefix=collaboration_args.experiment_prefix,
-        compression=hivemind.Float16Compression(),
-        batch_size_per_step=total_batch_size_per_step,
-        bandwidth=collaboration_args.bandwidth,
+        run_id=collaboration_args.experiment_prefix,
         target_batch_size=adjusted_target_batch_size,
+        batch_size_per_step=total_batch_size_per_step,
+        optimizer=opt,
+        scheduler=scheduler,
+        grad_compression=Float16Compression(),
+        state_averaging_compression=Float16Compression(),
+        averager_opts={"bandwidth": collaboration_args.bandwidth, **asdict(averager_args)},
```

Comment:
Can you please reorder these args to the same order as in the API reference?
https://learning-at-home.readthedocs.io/en/latest/modules/optim.html#module-hivemind.optim.collaborative

I'm trying to fix this order throughout the project, so it's easier to compare the sets of Optimizer args in different places.

Reply:
Could you also please check whether every argument from `arguments.py` goes to the right place?

Reply:
AFAIK they're already in the correct order: the link you attached refers to CollaborativeOptimizer, not Optimizer. As for checking the arguments, I launched the script and it appears to train correctly (at least no errors about unrecognized arguments)

Reply:
Actually, there were some errors and inconsistencies; fixed them now